### PR TITLE
Feature/state links

### PR DESCRIPTION
**Pull request by simonschwartz**
This PR add Links to state governments on the homepage. The closes SITES-481

### CODE REVIEW

**Comment by coderdan** [comment]
LGTM you'll need to rebase and squash

**Comment by tessereth** [comment]
please update changelog post 0.8.1 release

File: app/views/templates/root_node.html.haml [hunk 1]
```diff
@@ -44,18 +43,63 @@
...
+          %h3.state-title Tasmania
+          %a{href: 'http://www.tas.gov.au/'} www.tas.gov.au
+        %li.nt
+          %h3.state-title Northern Territory
+          %a{href: 'https://nt.gov.au/'} www.nt.gov.au
+        %li.act
+          %h3.state-title Australian Capital Territory
+          %a{href: 'http://www.act.gov.au/'} www.act.gov.au
+
+    /  %h3 Local Government
```
Comment (ltankey) [inline app/views/templates/root_node.html.haml]: We could probably remove these comments if they are not needed.

**Comment by maxious** [comment]
LGTM

**Pull request merged**